sv_mini_atlas/library/models.py

```diff
@@ -25,3 +31,42 @@ def __str__(self):
     @property
     def name(self):
         return self.metadata.get("work_title")
+
+    @classmethod
+    def dump_tree(cls, root=None, up_to=None, to_camel=True):
```

Comment:
This is great; I developed something a bit more specific for Digital Sirah that I could see refactoring a bit to make use of this pattern, but I think the fact that we got there independently on two different projects is a strong indicator that this an ATLAS feature that will get leveraged a lot.

refs:
- [GraphiQL query demonstrating collator](https://digital-sira-atlas-dev.herokuapp.com/graphql/#query=%7B%0A%20%20superheadings(first%3A%201)%20%7B%0A%20%20%20%20edges%20%7B%0A%20%20%20%20%20%20node%20%7B%0A%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20title%0A%20%20%20%20%20%20%20%20label%0A%20%20%20%20%20%20%20%20position%0A%20%20%20%20%20%20%20%20sectionCollations%0A%20%20%20%20%20%20%20%20sections(first%3A%201)%20%7B%0A%20%20%20%20%20%20%20%20%20%20edges%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20node%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20idx%0A%20%20%20%20%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A)
- ALTAS source code: https://github.com/eldarion-client/digital-sira-atlas/blob/cd793257b843c3843256504629c9167c8059b7c7/digital_sira_atlas/library/transformers.py#L1

- [Browsing the tree output](http://dii-demo.eldarion.com/browse?urn=0310Tabari.Tarikh.Shamela0009783-ara1.completed)
- Source code for component: https://github.com/eldarion-client/digital-sira/blob/2a925cf1c15ef7bf014f83351aee870885c36766/src/components/Browse/BrowseRow.vue